docs/_documentations/openshiftregistry.md

```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
       ```
 6. Copy the value from the **token** field.
 
-### Adding the OpenShift registry in Che
+### OpenShift registry in Che and Codewind
+
+#### Adding the OpenShift registry in Che
```

Comment:
Pls delete section `Adding the OpenShift registry in Che`, see bottom review comment where i merged this info